src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -398,6 +398,10 @@ In this message, the term "binding" means a piece of generated code that makes i
     <value>Using the DX DEX Compiler is deprecated. Please update `$(AndroidDexTool)` to `d8`.</value>
     <comment>The following are literal names and should not be translated: D8, DEX, `$(AndroidDexTool)`, `d8`</comment>
   </data>
+  <data name="XA1023_DOTNET" xml:space="preserve">
```

Comment:
It's not important at all, but this could use other capitalization options too, like `XA1023_dotnet`, for example. I have a hunch the all-caps might have been following `XA0119_AAB`, which is perfectly good, but the other resource names have some additional variety in capitalization like `XA0000_TargetFrameworkVersion_for_API`, so there's flexibility to use other capitalizations too if desired.